browsers.config.mjs

```diff
@@ -58,19 +53,16 @@ BrowserStack.config = {
   safari_minus2: { ...macOSConfig, browser: 'safari', os_version: 'Big Sur' },
 
   // Mobile Devices
-  ios:     { ...deviceConfig, browser: 'iphone',  device : 'iPhone 14',          os: 'ios',     os_version : '16' },
-  android: { ...deviceConfig, browser: 'android', device : 'Google Pixel 6',     os: 'android', os_version : '12.0' },
-  samsung: { ...deviceConfig, browser: 'samsung', device : 'Samsung Galaxy S22', os: 'android', os_version : '12.0' },
+  ios:     { ...deviceConfig, browser: 'iphone',  device: 'iPhone 14',          os: 'ios',     os_version : '16' },
+  android: { ...deviceConfig, browser: 'samsung', device: 'Samsung Galaxy S23', os: 'android', os_version : '13.0' },
```

Comment:
We can't remove android out of this list: it's the majority of browsers on mobile!
On the other hand, Samsung Internet is a fork of Chromium with a significant feature diff. It has ~4.5% market share worldwide. https://gs.statcounter.com/browser-market-share/mobile/worldwide

Although, UK government recommends testing against it, I think we should only add them to available browser list for now. https://technology.blog.gov.uk/2018/02/26/weve-added-samsung-internet-to-our-browser-testing-recommendations/

Reply:
Changed to android